app/views/metrics/index.html.erb

```diff
@@ -24,3 +24,16 @@
 <% end %>
 <%= render partial: 'all_prisons', locals: { dataset: @dataset } %>
 
+<%= render 'graph' %>
+
+<div  class="grid-row">
+  <% @prisons.in_groups_of(@prisons.size < 3 ? @prisons.size : (@prisons.size / 3), nil) do |prisons| %>
```

Comment:
Prisons are displayed using 3 columns. That's why. 
I will tidy this up in a presenter, this ugly.